app.js

```diff
@@ -1,17 +1,17 @@
-const express = require("express");
-const { getTopics, pathInvalid, getArticles } = require("./controllers/controllers");
-const app = express();
-app.use(express.json());
-
-app.get("/api/topics", getTopics);
-
-app.get("/api/articles", getArticles);
-
-app.use((err, req, res, next) => {
-  console.error(err);
-  res.status(500).send("Server Error!");
-});
-
-app.all("*", pathInvalid);
-
-module.exports = app;
+const express = require("express");
+const { getTopics, pathInvalid, getArticles } = require("./controllers/controllers");
+const app = express();
+app.use(express.json());

```

Comment:
Line 4 is not needed for get requests, try not to paste in code until you _know_ it's needed.